src/icons/currency-flags/currency-flags.ts

```diff
@@ -0,0 +1,77 @@
+// This file was generated by webpack
+export const flags = [
+  'AED',
```

Comment:
this file should be ignored eslint?

Reply:
You mean remove the eslint rule for this file?